src/index.css

```diff
@@ -12,11 +14,16 @@
     sans-serif;
   --font-family--monospace: source-code-pro, Menlo, Monaco, Consolas, 'Courier New', monospace;
 
+  --spacing--gap: 10px;
```

Comment:
Should I keep banging the "relative units" drum? I admit my current understanding is basically "always use relative units" - which is an ironically absolutist view - so if you have a better mindset you can share I'd appreciate it!

Reply:
It's good to bang that drum! But in this case, I don't _want_ our spacing gap to be relative to font size. (At least, not _this_ spacing gap. We may want others, such as between typographic blocks, to be relative.) If a user has increased their default font size for accessibility reasons, I don't want the gap around our panels to be any larger, and in fact doing so would be counterproductive as it would gobble up valuable real-estate.

[Josh Comeau has an excellent article](https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/) that formed my thinking about when to use fixed versus relative units. I don't always make the correct choice (see: the button styling), but in this case and a few others, the choice was intentional.

That said, _please_ do continue to call attention to them when you see them and if you suspect it's a time when relative units are preferable!

Reply:
Oh awesome, I'll take a look - thanks for sharing, @reefdog!